src/trace-api.js

```diff
@@ -179,6 +191,31 @@ TraceApiImplementation.prototype.createChildSpan = function(options) {
 };
 
 /**
+ * Generates a stringified trace context that should be set as the trace context
+ * header in a response to an incoming web request. This value is based on
+ * the trace context header value in the corresponding incoming request, as well
+ * as the result from the local trace policy on whether this request will be
+ * traced or not.
+ * @param {string} incomingTraceContext The trace context that was attached to
+ * the incoming web request, or null if the incoming request didn't have one.
+ * @param {boolean} isTraced Whether the incoming was traced. This is determined
+ * by the local tracing policy.
+ * @returns {string} If the response should contain the trace context within its
+ * header, the string to be set as this header's value. Otherwise, an empty
+ * string.
+ */
+TraceApiImplementation.prototype.getResponseTraceContext = function(
+    incomingTraceContext, isTraced) {
+  var traceContext = this.agent_.parseContextFromHeader(incomingTraceContext);
+  if (!traceContext) {
+    return '';
+  }
+  traceContext.options = traceContext.options && isTraced ? 1 : 0;
```

Comment:
Not sure which is more readable but `traceContext.options & isTraced` should also work here.